api/API.md

```diff
@@ -39,29 +39,34 @@ The root endpoint for all API requests for version 1.0 of the API is `/api/v1`
 
 ### RecrudescenceTest
 
-Given an input `.csv` file with [drug testing data in the proper format (TODO)](), return estimates of which patients specified in the file are reinfections or recrudescences.
+Given an input `.xlsx` file with [drug testing data in the proper format (TODO)](), return estimates of which patients specified in the file are reinfections or recrudescences.
 
-*Method(s):* `PUSH`
+*Method(s):* `POST`
 
 *URL Endpoint:* `/api/v1/recrudescences`
 
-#### PUSH
+#### POST
 
 **Input Parameters (URL):**
 
 -   (optional) `iterations` (integer): The number of iterations to run the algorithm for on the data; higher values are more accurate but take longer to complete. Defaults to 10000.
 
 **Input Parameters (Data):**
 
--   `file` (.csv file, FormData): The file containing the testing data to be processed.
+-   `file` (.xlsx file, FormData): The file containing the testing data to be processed.
+
+**Input Parameters (URL):**
+
+-   (optional) `locirepeat` (list of integers): List of integers representing loci repeats. Each loci repeat must be individually entered for the request. Please ensure the correct amount of values are entered.
```

Comment:
Locirepeats shouldn't be optional; it should be a required parameter (file can't be processed correctly without it)

Reply:
Also, here's a description from `define_alleles` about what locirepeats actually are: `a vector of length <number of loci> with the type of each locus (dinucleotide, trinucleotide, etc. repeats)`

Reply:
Done, and thanks for the description! It is much better than the one I had in there